eng/build.ps1

```diff
@@ -178,7 +178,7 @@ function Process-Arguments() {
 
   if ($testVsi) {
     # Avoid spending time in analyzers when requested, and also in the slowest integration test builds
-    $script:skipAnalyzers = $true
+    $script:runAnalyzers = $false
```

Comment:
This is no longer necessary.

Reply:
I felt the comment above meant to have this explicitly specified, so the behavior is preserved even if the default for `runAnalyzers` changes in future.